### PR TITLE
Update electron

### PR DESCRIPTION
**Pull request by zancas**
This is @skyl 's fix to the electron dep.

### CODE REVIEW

File: native/Cargo.toml [hunk 1]
```diff
@@ -15,5 +15,5 @@ default-features = false
 features = ["napi-6"]
 
 [dependencies]
-zecwalletlitelib = { git = "https://github.com/zingolabs/zecwalletlitelib", branch = "master" }
+zecwalletlitelib = { path = "../../../zecwalletlitelibs/dev/" }
```
Comment (AloeareV) [inline native/Cargo.toml:18]: This doesn't look like it wants to be on dev!

File: package.json [hunk 2]
```diff
@@ -142,7 +142,7 @@
     "@types/url-parse": "^1.4.3",
     "cargo-cp-artifact": "^0.1",
     "concurrently": "^5.3.0",
-    "electron": "^13.1.2",
+    "electron": "18.2.4",
```
Comment (AloeareV) [inline package.json:145]: If this change just works out of the box, that's awesome!

**Review by AloeareV — CHANGES_REQUESTED** [review]

File: native/Cargo.toml (hunk 1 above)
Comment (juanky201271) [reply native/Cargo.toml:18]: Definitely this doesn't work in our repo.

**Review by juanky201271 — CHANGES_REQUESTED** [review]

File: native/Cargo.toml (hunk 1 above)
Comment (zancas) [reply native/Cargo.toml:18]: This is my mistake.   Fixing...

Comment (zancas) [reply native/Cargo.toml]: This is my mistake.   Fixing...

**Review by AloeareV — APPROVED** [review]
Great!

**Pull request merged**